docs/2. Product Features/08. HSM Templates.md

```diff
@@ -1,18 +1,36 @@
-**How to create HSM templates in Glific and use them in communication?** _4 Minutes Read_
+> ### **6 minutes read         _____________           `Beginner`**
 
-## Create an HSM template and Submit for WhatsApp approval
+HSM (Highly Structured Message) templates are pre-approved messages by Whatsapp which are used to send to the users when their session window is closed (i.e after 24hours of inactive conversation). These templates ensure compliance with WhatsApp's guidelines and allow NGO's to send notifications, customer support messages and alerts with placeholders for personalized information
+<br>
```

Comment:
All tags needs to close in the documentation markdowns:
<br />.

Update all <br> tags or check for any alternative to add spaces